indico/modules/events/abstracts/forms.py

```diff
@@ -124,6 +127,43 @@ def data(self):
         return data
 
 
+class AbstractJudgmentForm(IndicoForm):
+    """Form for judging an abstract"""
+
+    judgment = IndicoEnumSelectField(_("Judgment"), [DataRequired()], enum=AbstractAction,
+                                     skip={AbstractAction.change_tracks})
+    accepted_track = QuerySelectField(_("Track"), [HiddenUnless('judgment', AbstractAction.accept)],
+                                      get_label=lambda x: x.title, allow_blank=True,
+                                      description=_("The track this abstract will be accepted in"))
+    accepted_contrib_type = QuerySelectField(_("Contribution type"), [HiddenUnless('judgment', AbstractAction.accept)],
+                                             get_label=lambda x: x.name.title(), allow_blank=True,
+                                             description=_("The type of contribution that will be created"))
```

Comment:
I think I made a mistake. It's probably more correct to say `... will be converted into...` instead of `to`.